operator/controller/dnsnameresolver/controller.go

```diff
@@ -163,12 +161,12 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	// event then this will ensure that the resolver sends the resolution request
 	// for the DNS name, so that the status of the object gets updated with the
 	// corresponding IP addresses.
-	r.resolver.Add(dnsName, []ocpnetworkv1alpha1.DNSNameResolverResolvedAddress{}, matchesRegular, request.Name)
+	r.resolver.AddDNSNameResolverResolvedName(dnsName, []ocpnetworkv1alpha1.DNSNameResolverResolvedAddress{}, matchesRegular, request.Name)
```

Comment:
Can the name be more concise? Something like
```suggestion
	r.resolver.AddResolvedName(dnsName, []ocpnetworkv1alpha1.DNSNameResolverResolvedAddress{}, matchesRegular, request.Name)
```

Or even shorter:
```suggestion
	r.resolver.AddName(dnsName, []ocpnetworkv1alpha1.DNSNameResolverResolvedAddress{}, matchesRegular, request.Name)
```